src/server/middleware/redirects.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction} from 'express';
+
+export const redirects = {
+    redirectSignin: function(req: Request, res: Response, next: NextFunction): void { //redirect unauthenticated users
```

Comment:
I think you can combine these two functions into one. And then in the routes just make `router.use(redirect)` and remove them from every route declaration